modules/libcom/src/osi/epicsStdio.h

```diff
@@ -52,23 +83,72 @@ extern "C" {
 #  define putchar epicsStdoutPutchar
 #endif
 
+/**
+ * \brief `epicsSnprintf` is meant to have the same semantics as the C99 function `snprintf`
+ *
+ * \details
```

Comment:
Can the duplication of `\details` be avoided?  Maybe by moving to the `\file` block and linking from individual functions?  (cf `\ref` and `\section` an similar)